### PR TITLE
701: Java quickstarts cleanup:

### PR DESCRIPTION
**Pull request by bibryam**
Signed-off-by: Bilgin Ibryam <bibryam@gmail.com>

# Description

- Removed unused imports
- Code style: add space after statements
- Removed unused test code
- Moved Dapr client creation outside of loops
- Update dapr-sdk versions to 1.5
- Added Maven artifact name and description where missing
- Aligned Maven artifact names to camel case

## Issue reference

https://github.com/dapr/quickstarts/issues/701

Please reference the issue this PR will close: #_[701]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] The quickstart code compiles correctly
* [x] You've tested new builds of the quickstart if you changed quickstart code
* [x] You've updated the quickstart's README if necessary
* [x] If you have changed the steps for a quickstart be sure that you have updated the automated validation accordingly. All of our quickstarts have annotations that allow them to be executed automatically as code. For more information see [mechanical-markdown](https://github.com/dapr/mechanical-markdown#readme). For user guide with examples see [Examples](https://github.com/dapr/mechanical-markdown/tree/main/examples#mechanical-markdown-by-example).

### CODE REVIEW

**Comment by bibryam** [comment]
These code changes also require changes in the docs. The corresponding PR in the docs is here docs/pull/2639

File: bindings/java/http/batch/src/main/java/com/service/controller/BatchProcessingServiceController.java [hunk 1]
```diff
@@ -105,28 +103,15 @@ private Orders loadOrdersFromFile(String path) throws Exception {
...
-            String json = String.join(System.getProperty("line.separator"), 
-            "{\"orders\": [",
-            "{\"orderid\": 1, \"customer\": \"John Smith\", \"price\": 100.32},",
-            "{\"orderid\": 2, \"customer\": \"Jane Bond\", \"price\": 15.4},",
-            "{\"orderid\": 3, \"customer\": \"Tony James\", \"price\": 35.56}",
-            "]",
-            "}");
-
-            return mapper.readValue(json, Orders.class);
+        try (InputStream is = getClass().getClassLoader().getResourceAsStream(path)) {
```
Comment (paulyuk) [inline bindings/java/http/batch/src/main/java/com/service/controller/BatchProcessingServiceController.java:106]: removing mock is ok.  I agree with this.

File: pub_sub/java/sdk/checkout/src/main/java/com/service/CheckoutServiceApplication.java [hunk 2]
```diff
@@ -14,11 +14,11 @@ public class CheckoutServiceApplication {
 	public static void main(String[] args) throws InterruptedException{
 		String TOPIC_NAME = "orders";
 		String PUBSUB_NAME = "orderpubsub";
+		DaprClient client = new DaprClientBuilder().build();
```
Comment (paulyuk) [inline pub_sub/java/sdk/checkout/src/main/java/com/service/CheckoutServiceApplication.java:17]: good catch.  thank you.

**Review by paulyuk — CHANGES_REQUESTED** [review]
Thank you much @bibryam -- this brings good idiomatic changes for Java.  There's just one minor change suggested and then I'll take it.

File: bindings/java/http/batch/pom.xml [hunk 3]
```diff
@@ -10,8 +10,8 @@
 	<groupId>com.service</groupId>
 	<artifactId>BatchProcessingService</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
-	<name>OrderProcessingService</name>
-	<description>Demo for Dapr pubsub component</description>
+	<name>BatchProcessingService</name>
+	<description>Quickstart for Dapr Bingings building block</description>
```
[the remark this answers is not in the document]
Comment (bibryam) [reply bindings/java/http/batch/pom.xml]: thanks, this is fixed now. Notice also the related quickstart docs PR in the docs project :)

**Review by paulyuk — APPROVED** [review]
LGTM.  thank you for this @bibryam !

**Pull request merged**